examples/alias/main.tf

```diff
@@ -100,3 +100,52 @@ module "alias_existing" {
   }
 
 }
+
+module "sqs_events" {
+  source  = "terraform-aws-modules/sqs/aws"
+  version = "v3.3.0"
+
+  name = "${random_pet.this.id}-events"
+}
+
+module "lambda_function_event_mapping" {
+  source = "../../"
+
+  function_name = "${random_pet.this.id}-lambda-event-mapping"
+  handler       = "index.lambda_handler"
+  runtime       = "python3.8"
+  publish       = true
+
+  source_path = "${path.module}/../fixtures/python3.8-app1"
+  hash_extra  = "yo"
+
+  provisioned_concurrent_executions = 1
+
+  create_async_event_config    = true
+  maximum_event_age_in_seconds = 100
+
+  attach_policies = true
+  policies = [
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole",
+  ]
+  number_of_policies = 1
+}
+
+module "alias_no_refresh_event_mapping" {
+  source = "../../modules/alias"
+
+  create        = true
+  refresh_alias = false
+
+  name = "current-no-refresh"
+
+  function_name    = module.lambda_function_event_mapping.lambda_function_name
+  function_version = module.lambda_function_event_mapping.lambda_function_version
+
+  event_source_mapping = {
```

Comment:
Add this argument to other modules called in this example (alias_no_refresh, alias_refresh, alias_existing), and update their outputs accordingly. There is no need to have module "lambda_function_event_mapping" and "alias_no_refresh_event_mapping".

Reply:
Updated existing aliases with event source mapping